code/modules/overmap/overmap_object.dm

```diff
@@ -23,6 +23,11 @@
 	var/last_burn = 0                   // worldtime when ship last acceleated
 	var/burn_delay = 1 SECOND           // how often ship can do burns
 
+	var/list/comms_masers
```

Comment:
Should use the overmap register/unregister proc instead of these lists, I never got around to it on my branch, remind me to show you.